Src/Support/Google.Apis.Auth/Google.Apis.Auth.csproj

```diff
@@ -0,0 +1,69 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- nupkg information -->
+  <Import Project="..\CommonPackageInfo.xml" />
+  <PropertyGroup>
+    <Title>Google APIs Client Library</Title>
+    <PackageTags>Google</PackageTags>
+    <Description>
+The Google APIs Client Library is a runtime client for working with Google services.
+
+This package includes auth components like user-credential, authorization code flow, etc. for making authenticated calls using the OAuth2 spec.
+
+Supported Platforms:
+- .NET Framework 4.5+
+- NetStandard1.3
+    </Description>
+  </PropertyGroup>
+
+  <!-- configurations; common -->
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\..\google.apis.snk</AssemblyOriginatorKeyFile>
+    <DebugType>pdbonly</DebugType>
```

Comment:
I was duplicating current behaviour. But agreed that portable is better. Changed